### DISCUSSION TITLE
ESP32 ibeacon not detecting

### DISCUSSION THREAD

**Discussion by Ramyras** [arendst/Tasmota#15626] [Q&A] [answered]
Hi all
I have ESP32 with tasmota32-bluetooth.bin version 11.1.0(bluetooth)
Setoption115 1 and ibeacon 1 is in place and can see some BLE devices:
18:28:36.965 MQT: tele/tasmota_22CF04/SENSOR = {"Time":"2022-05-14T18:28:36","IBEACON":{"MAC":"5CC1D7A02EDD","RSSI":-93,"STATE":"ON","PERSEC":0}} - it is actually not my device just some randomly detected that is in range.
I have Chines USB Beacon dongle that can be configured as ibeacon FeasyBeacon FSC-BP101E (I susspect it is with BLE 5.2) 
When I use ibeacon scanners on phone I can see this device but ESP32 not see it at all.
Anything special I missing there?
Can someone help understand the reason why

**Comment by ingrove** [comment]
I have the same question.  If I flash the ESP32 with ESPresense, I am able to see the iBeacon that my phone is transmitting via the android app Beacon Simulator.  Using Tasmota32-bluetooth.bin I am not able to see the same iBeacon (setoption115 1, and ibeacon 1).  I've recompiled with "#define USE_IBEACON_ESP32" included with the same result.  Is there another compiler option needed to get this to work?

**Comment by Ramyras** [comment, marked as answer]
anything?

**Reply by Jason2866** [reply]
```
iBeaconOnlyAliased 0
BLEAddrFilter 3
```

**Reply by Ramyras** [reply]
Yes exactly BLEAddrFilter 3 solved the issue. According documentation 3 is default value but appears that default is 0.
Other but actually out of toppic for this question - would be good to updete official documentation that everything works with precompiled tasmota32-bluetooth.bin and do not need any custom compilations for this particular case.

**Reply by Jason2866** [reply]
Is there anything mentioned suggesting this feature is not included? If we would describe every feature which is included the docu would be horrible big :-)

**Reply by Ramyras** [reply]
looks like you took me bit in wrong way. I had in mind documentation for custom compilation to have ibeacon working. Now documentation explicitly says to have ibeacon working nedd custom compilation  using USE_IBEACON_ESP32. With recent versions this is not required at all. So just need delete some not relevant anymore part of documentation.
